velox/common/base/BitUtil.h

```diff
@@ -958,6 +958,21 @@ inline void padToAlignment(
   }
 }
 
+#if defined __has_builtin
```

Comment:
It would be nice to document this function. Perhaps, use documentation from https://gcc.gnu.org/onlinedocs/gcc/Other-Builtins.html

```
Returns x with the order of the bytes reversed; for example, 0xaabb becomes 0xbbaa. Byte here always means exactly 8 bits.
```